statistics/handle/handle_hist.go

```diff
@@ -30,20 +31,27 @@ import (
 	"go.uber.org/zap"
 )
 
+// TableColumnIDCombina contains chan model.TableColumnID and Exit channel
```

Comment:
Combina is not a word, either Combine or Combined, TableColumnID should be enough?

Reply:
```TableColumnID``` have been used.

Reply:
> Combina is not a word, either Combine or Combined, TableColumnID should be enough?

Combine is not unsuitable.

Reply:
Fix